maven-build-caching-samples/animal-sniffer-project/pom.xml

```diff
@@ -0,0 +1,125 @@
+                  <artifactId>animal-sniffer-maven-plugin</artifactId>
+                  <executions>
+                    <execution>
+                      <id>animal-sniffer-check</id>
+                      <inputs>
+                        <ignoredProperties>
+                          <ignore>localRepository</ignore>
+                          <ignore>project</ignore>
+                        </ignoredProperties>
+                        <fileSets>
+                          <fileSet>
+                            <name>outputDirectory</name>
+                            <normalization>CLASSPATH</normalization>
+                          </fileSet>
+                          <fileSet>
+                            <name>testOutputDirectory</name>
+                            <normalization>CLASSPATH</normalization>
+                          </fileSet>
+                        </fileSets>
+                        <properties>
+                          <property>
+                            <name>ignores</name>
+                          </property>
+                          <property>
+                            <name>annotations</name>
+                          </property>
+                          <property>
+                            <name>ignoreDependencies</name>
+                          </property>
+                          <property>
+                            <name>includeDependencies</name>
+                          </property>
+                          <property>
+                            <name>excludeDependencies</name>
+                          </property>
+                          <property>
+                            <name>checkTestClasses</name>
+                          </property>
+                          <property>
+                            <name>skip</name>
+                          </property>
+                          <property>
+                            <name>failOnError</name>
+                          </property>
+                        </properties>
+                      </inputs>
+                      <nestedProperties>
+                        <property>
+                          <name>signature</name>
+                          <inputs>
+                            <properties>
+                              <property>
+                                <name>groupId</name>
+                              </property>
+                              <property>
+                                <name>artifactId</name>
+                              </property>
+                              <property>
+                                <name>version</name>
+                              </property>
+                            </properties>
+                          </inputs>
+                        </property>
+                      </nestedProperties>
+                      <outputs>
+                        <cacheableBecause>Animal Sniffer is slow</cacheableBecause>
```

Comment:
haha, maybe a different reason would be better here? like it has proper input and outputs and is deterministic?